perun-openapi/openapi.yml

```diff
@@ -7372,6 +7372,20 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /urlinjsonout/usersManager/getUsersBySpecificUser:
```

Comment:
The URL should be '/json/usersManager/getUsersBySpecificUser' because the method is GET type and the parameter is just a user id.